src/main/webapp/index.html

```diff
@@ -278,14 +298,32 @@ <h4>413 Advising List</h4>
                 <div class="mdl-card__supporting-text">
                     <h4>Graduation List</h4>
                     <p>Compile a list of all students who have graduated. A future version will
-                      allow to specify a date range.</p>
+                        allow to specify a date range.</p>
                     <button class="mdl-button mdl-js-button mdl-button--fab" onclick="return generateList(this, 'graduated')">
                         <i class="material-icons">forward</i>
                     </button>
                 </div>
             </section>
         </div>
     </main>
+    <div class="mdl-layout__tab-panel" id="senior_advising_read_only" style="display: none">
```

Comment:
I suggest you remove all this HTML and decide on the server side (based on which password the user typed in) whether you send the editable or read-only HTML. I don't see the need for duplicating this HTML.